pkg/monitor/alerts.go

```diff
@@ -48,6 +61,14 @@ func whenWasAlertInState(ctx context.Context, prometheusClient prometheusv1.API,
 		return nil, err
 	}
 
+	if namespace == platformidentification.NamespaceOther {
```

Comment:
Do you want this to say `if namespace == platformidentification.NamespaceOther || len(namespace) == 0`? since the query above is identical for those two cases?